policy/modules/kernel/storage.te

```diff
@@ -16,6 +16,8 @@ attribute storage_unconfined_type;
 # /dev/hd* and /dev/sd*.
 #
 type fixed_disk_device_t;
+# nvme_device_t was merged into fixed_disk_device_t in Fedora 34+
```

Comment:
It was actually not merged but rather renamed.

How about direct statement "will be removed in f36"? If we had some template for changes like this, we could look for them more easily at the right time for them.

Reply:
> It was actually not merged but rather renamed.

At a high level it was merged into the existing `fixed_disk_device_t` type. But I could change it to "removed in favor" as in the commit message.

> How about direct statement "will be removed in f36"? If we had some template for changes like this, we could look for them more easily at the right time for them.

IMHO, "obsoleted in F34" already gives enough information to know when to remove it, given a strategy. I think it would be better to not hardcode the strategy itself in the comments, just the "input data".

Reply:
Agree this is better.